orbit/cmd/desktop/desktop.go

```diff
@@ -110,6 +111,19 @@ func main() {
 		log.Info().Msgf("got a TUF update root: %s", tufUpdateRoot)
 	}
 
+	if runtime.GOOS == "linux" {
```

Comment:
Let's add a comment why we do this for Linux only (mainly because that's where the issue was reported).